packages/venia-concept/src/components/ProductStickyFooter/productStickyFooter.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+import classify from 'src/classify';
+import defaultClasses from './productStickyFooter.css';
+import Icon from 'src/components/Icon';
+
+const SUCCESS_STATE_LIFETIME = 3000;
+
+class ProductStickyFooter extends Component {
+    static propTypes = {
+        classes: PropTypes.shape({
+            root: PropTypes.string,
+            addToCart: PropTypes.string,
+            addToCartInProgress: PropTypes.string,
+            addToCartSuccess: PropTypes.string,
+            loadingContainer: PropTypes.string
+        }),
+        addToCartHandler: PropTypes.func,
+        addToCartError: PropTypes.bool,
+        addToCartActiveRequest: PropTypes.bool
+    };
+
+    state = {
+        addToCartButtonSuccessState: false
+    };
+
+    componentWillUnmount() {
+        clearTimeout(this.resetButtonTimer);
+    }
+
+    get getSuccessButtonData() {
```

Comment:
Please either drop the `get` prefix from these getters or make them class functions (not getters).